fms_parser.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/PuerkitoBio/goquery"
+	"os"
+	"strings"
+)
+
+// func main() {
```

Comment:
Maybe this could be a test function?
https://blog.alexellis.io/golang-writing-unit-tests/
https://www.golang-book.com/books/intro/12
etc